### PR TITLE
ML - Fixed script formatting for prediction

### LINKED ISSUE #11619: ML Extension - PREDICT sql script formatted for readability
**Is your feature request related to a problem? Please describe.**
The PREDICT function is not preformatted, making it hard to read. 

**Describe the solution or feature you'd like**
SQL Keywords should be in capital. 
Indentation to make it easier to read. 
Here's an example: 
[expected_predict_format.txt](https://github.com/microsoft/azuredatastudio/files/5018600/expected_predict_format.txt)

**Describe alternatives you've considered**
Formatting on my own. Or perhaps use one of sql formatter extension.

**Additional context**
n/a

### PR DESCRIPTION
**Pull request by llali**
This PR fixes #11619

### CODE REVIEW

**Comment by MsSQLGirl** [comment]
Looks good :)

File: extensions/machine-learning/src/prediction/predictService.ts [hunk 1]
```diff
@@ -173,7 +173,7 @@ WITH predict_input
 AS (
 	SELECT TOP 1000
 	${this.getInputColumnNames(columns, 'pi')}
-	FROM [${utils.doubleEscapeSingleBrackets(sourceTable.databaseName)}].[${sourceTable.schema}].[${utils.doubleEscapeSingleBrackets(sourceTable.tableName)}] as pi
+	FROM [${utils.doubleEscapeSingleBrackets(sourceTable.databaseName)}].[${sourceTable.schema}].[${utils.doubleEscapeSingleBrackets(sourceTable.tableName)}] AS pi
```
Comment (udeeshagautam) [inline extensions/machine-learning/src/prediction/predictService.ts:176]: I keep thinking is there no way to take out these queries and have them in sperate const or so to take it away from the service code...?

Comment (udeeshagautam) [reply extensions/machine-learning/src/prediction/predictService.ts:176]: not a PR blocker but for later...might be easy to understand and update the code...

Comment (llali) [reply extensions/machine-learning/src/prediction/predictService.ts:176]: sure I can move them to a different module

**Pull request merged**